rekognition_tutorial/short_sa_rekognition_tutorial.ipynb

```
@@ -0,0 +1,1222 @@
+    "        print(f\"{len(changes['GroundTruth'])} to add\")\n",
+    "        rek_client.update_dataset_entries(Changes = changes,\n",
+    "                                          DatasetArn = dataset_arn)\n",
+    "        print(f\"Updated dataset {dataset_arn}\")\n",
+    "        finished = False\n",
+    "        while finished == False:\n",
+    "\n",
+    "            dataset = rek_client.describe_dataset(DatasetArn = dataset_arn)\n",
+    "\n",
+    "            status = dataset['DatasetDescription']['Status']\n",
+    "            status_message = dataset['DatasetDescription']['StatusMessage']\n",
+    "            \n",
+    "            if status == \"UPDATE_IN_PROGRESS\":\n",
+    "                print((f\"Updating dataset: {dataset_arn} \"))\n",
+    "                time.sleep(5)\n",
+    "                continue\n",
+    "\n",
+    "            if status == \"UPDATE_COMPLETE\":\n",
+    "                print(f\"Dataset updated: {status} : {status_message} : {dataset_arn}\")\n",
+    "                finished=True\n",
+    "                continue\n",
+    "\n",
+    "            if status == \"UPDATE_FAILED\":\n",
+    "                print(f\"Dataset update failed: {status} : {status_message} : {dataset_arn}\")\n",
+    "                raise Exception (f\"Dataset update failed: {status} : {status_message} : {dataset_arn}\")\n",
+    "                \n",
+    "\n",
+    "            print(f\"Failed. Unexpected state for dataset update: {status} : {status_message} : {dataset_arn}\")\n",
+    "            raise Exception(f\"Failed. Unexpected state for dataset update: {status} : {status_message} :{dataset_arn}\")\n",
+    "            \n",
+    "        print(f\"Added entries to dataset\")\n",
+    "        \n",
+    "        return status, status_message\n",
+    "   \n",
+    "    \n",
+    "    except ClientError as err:  \n",
+    "        print(f\"Couldn't update dataset: {err.response['Error']['Message']}\")\n",
+    "        raise"
```

Maybe raise an exception too?